src/snmpd/patch-5.7.3+dfsg/series

```diff
@@ -1,4 +1,5 @@
 0001-SNMP-Stop-spamming-logs-with-statfs-permission-denie.patch
 0002-at.c-properly-check-return-status-from-realloc.-Than.patch
-0003-CHANGES-BUG-2743-snmpd-crashes-when-receiving-a-GetN.patch
+#0003-CHANGES-BUG-2743-snmpd-crashes-when-receiving-a-GetN.patch
```

Comment:
># [](start = 0, length = 1)

You can remove this line, and delete the patch file.

Reply:
Sure. ll do it.